### PR TITLE
adding soft accuracy score

### PR DESCRIPTION
**Pull request by kegl**
I need a rapid review on this.

Travis is down due to the scipy problem, but pytest runs fine on my computer.

### CODE REVIEW

File: rampwf/score_types/soft_accuracy.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""A generalization of the classification accuracy with cross-class scores.
+
+Soften the accuracy score by giving scores through certain msiclassifications
```
Comment (agramfort) [inline rampwf/score_types/soft_accuracy.py]: msiclassifications -> misclassifications

File: rampwf/score_types/soft_accuracy.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+defined by the score matrix. For example, in ordinal regression we may want
+not to penalize too much misclassifications to neighbor classes. The score also
+generalizes RMSE-like regression scores for ordinal regression (when true and
+predicted output levels are coming from a fixed set) by allowing to defines
```
Comment (agramfort) [inline rampwf/score_types/soft_accuracy.py]: defines -> define

File: rampwf/score_types/tests/test_soft_accuracy.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+y_true_proba_2 = np.array([0, 1, 0])
+y_true_proba_3 = np.array([0.5, 0.5, 0])
+
+y_proba_1 = np.array([1, 0, 0])
+y_proba_2 = np.array([0, 1, 0])
+
+
+def test_soft_accuracy():
+    score_1 = SoftAccuracy(score_matrix=score_matrix_1)
+    assert score_1(np.array([y_true_proba_1]), np.array([y_proba_1])) == 1
```
Comment (agramfort) [inline rampwf/score_types/tests/test_soft_accuracy.py]: no not of np.array they are already arrays

Comment (kegl) [reply rampwf/score_types/tests/test_soft_accuracy.py]: [y_true_proba_1] would be just a list of np.arrays, no?

**Pull request merged**